### PR TITLE
Add a test for minBufferBindingSize

### PR DESCRIPTION
**Pull request by Richard-Yunchao**

### CODE REVIEW

**Comment by Richard-Yunchao** [comment]
PTAL. 
@kainino0x and @Kangz and @austinEng , how can I add minBufferBindingSize into node_modules/@webgpu/types/dist/index.d.ts?

**Comment by austinEng** [comment]
Please make a PR here https://github.com/gpuweb/types, and then Kai or I have to publish it to NPM

**Comment by Richard-Yunchao** [comment]
> Please make a PR here https://github.com/gpuweb/types, and then Kai or I have to publish it to NPM

Thank you Austin. 

Now that the change has been made in that project. How could I re-run the CI? BTW, please take a look at this pr when you have time. If this one is OK, I will add tests for a few other parameters which must be undefined or false under some situations.

**Comment by kainino0x** [comment]
you'll have to update the `@webgpu/types` dependency in package.json to get the new types. I just published 0.0.26 so you can update to that version.

**Comment by kainino0x** [comment]
I think I messed up that release, use 0.0.27 instead.

**Comment by Richard-Yunchao** [comment]
The CI failed again. Looking into the failure info, I suppose that we need to update package-lock.json too? But I tried at my local machine, it didn't work (I only changed the version from 0.0.24 to 0.0.27 for `@webgpu/types` in package-lock.json).

**Comment by austinEng** [comment]
You should update `package.json` and then run `npm install` again. That will automatically update `package-lock.json`

**Comment by kainino0x** [comment]
oh yes, sorry, forgot about that step.

File: src/webgpu/api/validation/createBindGroupLayout.spec.ts [hunk 1]
```diff
@@ -64,6 +64,33 @@ g.test('visibility_and_dynamic_offsets')
...
+  .fn(async t => {
+    const { type, minBufferBindingSize } = t.params;
+
+    let success = false;
+    if (minBufferBindingSize > 0) {
+      if (
+        type === 'uniform-buffer' ||
+        type === 'storage-buffer' ||
+        type === 'readonly-storage-buffer'
+      ) {
```
Comment (kainino0x) [inline src/webgpu/api/validation/createBindGroupLayout.spec.ts:82]: optional nit:
```suggestion
      if (type in kBufferBindingTypeInfo) {
```

File: src/webgpu/api/validation/createBindGroupLayout.spec.ts [hunk 2]
```diff
@@ -64,6 +64,33 @@ g.test('visibility_and_dynamic_offsets')
     }, !success);
   });
 
+g.test('mininum_buffer_binding_size')
+  .params(
+    params()
+      .combine(poptions('type', kBindingTypes))
+      .combine(poptions('minBufferBindingSize', [0, 4]))
```
Comment (kainino0x) [inline src/webgpu/api/validation/createBindGroupLayout.spec.ts:71]: I think we should test `undefined` here in order to test the default value of this option.
```suggestion
      .combine(poptions('minBufferBindingSize', [undefined, 0, 4]))
```

Most of the time I think these will go in operation tests, but in this case it only really affects validation.

**Comment by kainino0x** [comment]
Thanks!

**Comment by Richard-Yunchao** [comment]
Thanks for your review, @kainino0x and @austinEng !

**Pull request merged**